config/routes.js

```diff
@@ -10,10 +22,46 @@ module.exports = server => {
 
 function register(req, res) {
   // implement user registration
-}
+  const creds = req.body;
+  const hash = bcrypt.hashSync(password, 14);
+  req.body.password = hash;
```

Comment:
We're hashing the password on req.body, but sending `creds` to the database with an unhashed password. Make sure to keep an eye on consistency when you assign req.body to a variable!